packages/substrate/package.json

```diff
@@ -55,7 +55,7 @@
     "typescript": "^5.4.0"
   },
   "dependencies": {
-    "@buildwithsygma/core": "workspace:^",
+    "@buildwithsygma/core": "1.0.0",
```

Comment:
is this going to work with workspaces? in the widget react package still defines the dep as workspace